src/renderer/services/file-management-system/Md5Hasher.ts

```diff
@@ -1,100 +1,46 @@
-import * as CryptoJS from "crypto-js";
-
-/**
- * Abstract hasher template.
- * Provides this project with a type for CryptoJS.algo.MD5 from CryptoJS which does not export this type.
- */
- interface Hasher {
-    /**
-     * The number of 32-bit words this hasher operates on. Default: 16 (512 bits)
-     */
-    blockSize: number;
-    /**
-     * Resets this hasher to its initial state.
-     */
-    reset(): void;
-    /**
-     * Updates this hasher with a message.
-     *
-     * @param messageUpdate The message to append.
-     *
-     * @return This hasher.
-     */
-    update(messageUpdate: CryptoJS.lib.WordArray | string): this;
-    /**
-     * Finalizes the hash computation.
-     * Note that the finalize operation is effectively a destructive, read-once operation.
-     *
-     * @param messageUpdate (Optional) A final message update.
-     *
-     * @return The hash.
-     */
-    finalize(messageUpdate?: CryptoJS.lib.WordArray | string): CryptoJS.lib.WordArray;
-}
+import * as SparkMD5 from "spark-md5";
```

Comment:
I think you only use the `ArrayBuffer` so you could just import that like so:
```
import { ArrayBuffer } from "spark-md5";
```

Reply:
Done.